src/ccl_background.c

```diff
@@ -17,127 +17,119 @@ INPUT: scale factor, cosmology
 TASK: Compute E(a)=H(a)/H0
 */
 static double h_over_h0(double a, ccl_cosmology * cosmo, int *status)
-{ 
-  // Check if massive neutrinos are present - if not, we don't need to compute their contribution
+{
+  // Check if massive neutrinos are present - if not, we don't need to
+  // compute their contribution
   double Om_mass_nu;
   if ((cosmo->params.N_nu_mass)>1e-12) {
-    Om_mass_nu = ccl_Omeganuh2(a, cosmo->params.N_nu_mass, cosmo->params.mnu, cosmo->params.T_CMB, cosmo->data.accelerator, status) / (cosmo->params.h) / (cosmo->params.h);
+    Om_mass_nu = ccl_Omeganuh2(
+      a, cosmo->params.N_nu_mass, cosmo->params.mnu, cosmo->params.T_CMB,
+      cosmo->data.accelerator, status) / (cosmo->params.h) / (cosmo->params.h);
     ccl_check_status(cosmo, status);
   }
   else {
     Om_mass_nu = 0;
   }
 
-  return sqrt((cosmo->params.Omega_m+cosmo->params.Omega_l*pow(a,-3*(cosmo->params.w0+cosmo->params.wa))*exp(3*cosmo->params.wa*(a-1))+cosmo->params.Omega_k*a+(cosmo->params.Omega_g + cosmo->params.Omega_n_rel)/a + Om_mass_nu*a*a*a)/(a*a*a));
+  /* Calculate h^2 using the formula (eqn 2 in the CCL paper):
+    E(a)^2 = Omega_m a^-3 +
+             Omega_l a^(-3*(1+w0+wa)) exp(3*wa*(a-1)) +
+             Omega_k a^-2 +
+             (Omega_g + Omega_n_rel) a^-4 +
+             Om_mass_nu
+  */
+  return sqrt(
+    (cosmo->params.Omega_m +
+     cosmo->params.Omega_l *
+       pow(a,-3*(cosmo->params.w0+cosmo->params.wa)) *
+       exp(3*cosmo->params.wa*(a-1)) +
+     cosmo->params.Omega_k * a +
+     (cosmo->params.Omega_g + cosmo->params.Omega_n_rel) / a +
+     Om_mass_nu * a*a*a) / (a*a*a));
 }
 
-/* --------- ROUTINE: ccl_rho_x ---------
+/* --------- ROUTINE: ccl_omega_x ---------
 INPUT: cosmology object, scale factor, species label
-TASK: Compute rho_x(a), with x defined by species label.
+TASK: Compute the density relative to critical, Omega(a) for a given species.
 Possible values for "label":
 ccl_species_crit_label <- critical (physical)
-ccl_species_m_label <- matter (physical)
-ccl_species_l_label <- DE (physical)
-ccl_species_g_label <- radiation (physical)
-ccl_species_k_label <- curvature (physical)
-ccl_species_ur_label <- massless neutrinos (physical)
-ccl_species_nu_label <- massive neutrinos (physical)
+ccl_species_m_label <- matter
+ccl_species_l_label <- DE
+ccl_species_g_label <- radiation
+ccl_species_k_label <- curvature
+ccl_species_ur_label <- massless neutrinos
+ccl_species_nu_label <- massive neutrinos
 */
-double ccl_rho_x(ccl_cosmology * cosmo, double a, ccl_species_x_label label, int is_comoving, int *status)
+double ccl_omega_x(ccl_cosmology * cosmo, double a, ccl_species_x_label label, int *status)
```

Comment:
This is an API change, it should be documented in the CHANGELOG. Checks pass, so I assume this was corrected in the python wrapping.

Reply:
Oh, wait, did omega_x and rho_x simply switch order in the file?

Reply:
yes. :)